src/styles/styles.js

```diff
@@ -739,6 +739,42 @@ const styles = {
         justifyContent: 'center',
     },
 
+    textInputEmojiSearch: addOutlineWidth({
+        backgroundColor: themeColors.componentBG,
+        borderColor: themeColors.border,
+        color: themeColors.text,
+        fontFamily: fontFamily.GTA,
+        fontSize: variables.fontSizeNormal,
+        borderWidth: 0,
+        borderRadius: 0,
+        height: 'auto',
+
+        // On Android, multiline TextInput with height: 'auto' will show extra padding unless they are configured with
+        // paddingVertical: 0, alignSelf: 'center', and textAlignVertical: 'center'
+
+        paddingHorizontal: 8,
+        marginVertical: 5,
+        paddingVertical: 0,
+        textAlignVertical: 'center',
+    }, 0),
+
+    emojiHeaderStyle: {
+        backgroundColor: themeColors.hoverComponentBG,
+        fontWeight: 'bold',
```

Comment:
You'll probably need to pull in the correct fontFamily for bold fonts too. 

```
fontFamily: fontFamily.GTA_BOLD,
fontWeight: fontWeightBold,
```

Reply:
Cool, thanks! This is still very. Much WIP btw but I'll tag you in to double check you like how it looks when it's closer to being done

Reply:
Cool cool, definitely want to give you space to work on this but figured I'd leave that little hot tip. Can't wait for this one!